doc/vxlan/Overlay ECMP with BFD.md

```diff
@@ -232,20 +233,98 @@ VnetOrch shall create an entry in STATE_DB for the active overlay routes eligibl
 
 ```
 STATE_DB|ADVERTISE_NETWORK_TABLE|{{ip_prefix}}
+    "profile": {{profile_name}}
 ```
 
 The above entry shall be subscribed for by bgpcfgd and advertised by the “network” command. 
+Notes: Currently, only one profile_name is supported, the value is fixed as "FROM_SDN_SLB_ROUTES" 
 
 For example:
 ```
 router bgp 1
  address-family ipv4 unicast
-  network 10.0.0.0/8
+  network 10.0.0.0/8  route-map FROM_SDN_SLB_ROUTES_RM
  exit-address-family
- ```
+```
+
+This configuration example says that network 10.0.0.0/8 will be announced to all neighbors. FRR bgpd doesn’t care about IGP routes when announcing its routes. 
+The profile would be transformed to route-map and associated with IP prefix. 
+
+VnetOrch shall create a profile in APP_DB, which would be associated to the IP prefix when advertised by "network" command. 
+
+```
+APPL_DB:BGP_PROFILE_TABLE:{{profile_name}}
+    "community_id": {{community_string}}
+```
+
+Notes: Same here, currently only one profile is supported - "FROM_SDN_SLB_ROUTES" 
```

Comment:
Let's not specify the BGP profile name here as it can be changed in future but we can keep the example below